packages/node_modules/@webex/redux-module-conversation/src/actions.js

```diff
@@ -438,7 +443,8 @@ export function handleAdaptiveCardSubmitAction(url, actionInput, parentId, spark
       {url},
       actionInput,
       {id: parentId}
-    ).then((activity) => {
-      dispatch(acknowledgeAdaptiveCardSubmitAction(activity));
-    });
+    ).then((activity) =>
+      dispatch(acknowledgeAdaptiveCardSubmitAction(activity.id, parentId)))
+      .catch(() =>
```

Comment:
Do you want to `catch` any failures from the `spark.internal.conversation.cardAction` promise? If so, this `catch` will not do so.

Reply:
Do you mean not getting the error message? 

If it fails in submitting action then I'm passing activityId and parentId as null. I'm checking [here](https://github.com/webex/react-widgets/pull/1054/files#diff-36a8138b14ba83ac03da30246f485bb2R94) to show `Unable to send the request` as a button state

Reply:
Ah ok, I was confused by the indentation of the `catch` statement. Could you align it with the above `then`?

```js
).then((activity) =>
  dispatch(acknowledgeAdaptiveCardSubmitAction(activity.id, parentId))
).catch(() =>
```